### ISSUE TITLE
view only members / non-voting members

### ISSUE THREAD

**Issue by TuniRR** [loomio/loomio#10764]
Hello,
Thanks so much for all the work you've all put into this, we're finding loomio to be a really versatile platform.

We've ran into an obstacle for our organisation though.
Basically we want to use loomio to host our quarterly agenda decisions, however we have associate members that don't have a vote.
We want to allow them to view the discussion without having to manually remove them from a decision.

If there'd be a way to flag a user as not voting to remove these permissions it would be really handy.

Perhaps in /admin

Thanks in advance, and apologies if this request is not clear enough.

**Comment by robguthrie** [comment]
Hi @TuniRR , I think the functionality you want already exists. You can make a poll "invite only" within a discussion that is visible to the whole group.

See this page: https://help.loomio.com/en/user_manual/polls/poll_templates/index.html
The image under the text "As usual, you can set a closing date for the proposal to give some time for people to respond." shows the option to select if the poll is "everyone in group" or "invited people only".

You can save this setting in a template for future use too.

Is that what you're looking for?

**Comment by TuniRR** [comment]
Hi @robguthrie , Thanks for your quick reply.

If there was a way to add which members get invited to a poll repeatedly then that would certainly work.
Our situation is we have quarterly gatherings with a fairly large amount of proposals and also quite a large amount of members to invite, we've found that this is making quite a lot of work load for our secretarial working group. With room of error.

We've tried to get to a workaround where members have something in their name, in our case (M) to signify being a full member.
However when we try to invite these members by searching for (M) they don't all come up.

Basically we're trying to find a way to streamline it to make less workload.

Again I really appreciate your time in replying to this.

**Comment by robguthrie** [comment]
Put all your voting people in a subgroup, then set the poll to be "invited people only" and invite the subgroup to be voters in the poll.

**Comment by TuniRR** [comment]
That's very smart!
I'll give that a go

**Comment by robguthrie** [comment]
let me know if that worked for you :)

**Comment by TuniRR** [comment]
I'll be testing out your solution this coming week and will definitely
report back.

Again thanks for your time.
